the following change to const is necessary since gcc-11

thanks to seahawk1986 @ vdr-portal.de
https://www.vdr-portal.de/forum/index.php?thread/134005-vdr-2-4-6-kompiliert-mit-gcc-11-0-0-nicht/&postID=1340496#post1340496

Signed-off-by: Martin Dummer <martin.dummer@gmx.net>

--- a/conflictcheck.h
+++ b/conflictcheck.h
@@ -73,7 +73,7 @@
 class TimerObjSort
 {
 public:
-    bool operator()(cConflictCheckTimerObj* a, cConflictCheckTimerObj* b) {
+    bool operator()(const cConflictCheckTimerObj* a, const cConflictCheckTimerObj* b) const {
         return (a->Compare(*b) < 0);
     }
 };
